app/overrides/spree/admin/shared/_order_summary/add_taxjar_reported_at.html.erb.deface

```diff
@@ -1,8 +1,30 @@
 
 <!-- insert_bottom "[id='order_tab_summary'] > .additional-info" -->
 
+<% content_for :head do %>
+  <style>
+    .pill-success {
+      background: #c8d8e8;
+    }
+  </style>
```

Comment:
I am surprised we need this. Bootstrap provides a `.pill-success` class:

https://getbootstrap.com/docs/4.0/components/badge/#pill-badges

Reply:
Huh!

This is where the pill classes come from in solidus! I don't think they use the bootstrap classes
https://github.com/solidusio/solidus/blob/master/backend/app/assets/stylesheets/spree/backend/components/_pills.scss

Reply:
Wild! All this time I thought we used Bootstrap in the admin for that kind of stuff.